test/content/timeout-issue.js

```diff
@@ -0,0 +1,37 @@
+const { test } = require('tap')
+
+function cleanCache (module) {
+  delete require.cache[require.resolve(module)]
+}
+
+function requireFresh (module) {
+  cleanCache(module)
+  return require(module)
+}
+
+test('timeout issue content', async t => {
+  t.afterEach(() => {
+    delete process.env.GITHUB_HOST
+    delete process.env.GITHUB_URL
+    cleanCache('../../lib/env')
+  })
+
+  t.test('includes the link to the initial branch on the custom github host', async t => {
+    process.env.GITHUB_HOST = 'https://enterprise.github/api/v3/'
```

Comment:
I’m a bit confused that `GITHUB_HOST` has a path here (`/api/v3/`), maybe we can call it `GITHUB_API_BASE_URL`? Can we document the new environment variable somewhere?